wasm_samples/Cargo.toml

```diff
@@ -23,8 +23,9 @@ members = [
     "executor_custom_data_model",
     "query_assets_and_save_cursor",
     "smart_contract_can_filter_queries",
-    "multisig_register",
-    "multisig",
+    "multisig_accounts",
+    "multisig_domains",
+    "multisig_transactions",
```

Comment:
I wonder if it is really necessary to have MS functionality to be spread across three smartcontracts? I wish all the logic to be contained within a single one. To support different "modes" of operation (i.e. as a domain initialiser, accounts registry, transactions registry) it can rely on various metadata.

Reply:
For clarity, they are hierarchical according to their authorities atm